tests/e2e/common.go

```diff
@@ -369,7 +369,11 @@ func writeToFile(resource Output) error {
 
 func closeAllFiles() error {
 	for _, file := range openFiles {
-		err := file.Close()
+		err := file.Sync()
+		if err != nil { // flush file to disk
+			return err
+		}
```

Comment:
I'm actually not sure if it's not flushed or not when calling `file.Close()` but this is just in case